Dockerfile

```diff
@@ -22,13 +22,15 @@ WORKDIR /usr/src/app
 #============
 FROM base as tsbuild
 
-# use buildin python to faster build
-RUN install-apt build-essential python3
-RUN npm install -g yarn-deduplicate
+RUN install-npm yarn-deduplicate
```

Comment:
Non global?

Reply:
`install-npm` is from buildpack and installs with npm globally, but now removed, as there is no dedupe whichout lockfile.